tests/lint.py

```diff
@@ -0,0 +1,99 @@
+"""Tests performing linter-like checks"""
```

Comment:
Maybe this should be called `custom_lint.py` or something like that, to differentiate against running some other linter?